.gitignore

```diff
@@ -105,3 +105,4 @@ venv.bak/
 node_modules
 package.json
 yarn.lock
+tensorflow-introduction/
```

Comment:
Hmm I am a bit surprised. Did you mean to have this ignored or not ? Because all the related files are already tracked